tools/format_release_notes.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ "$1" = "-h" ]
+  then
+    echo -e 'Grab the "Future Releases" section from the release notes and format it in markdown for GitHub releases
+
+Usage: tools/format_release_notes.sh'
+    exit 0
+fi
+evalml_version=$(sed -n -E "s/.*version=\'([0-9A-Za-z\.\_]+)\'.*/\1/p" setup.py)
+content_raw=$(sed -n -e "/^\*\*v0.16.1/,/^\*\*v0.16.0/p;/^\*\*v0.16.0/q" docs/source/release_notes.rst | sed '/^\.\. warning::$/d' | tail -n +2 | sed '$ d' | awk 'NF')
```

Comment:
This is like black magic to me 😂 But I tried to use it and it seems to always output v0.16.1--is that because this line hardcodes it?

Reply:
This could be:

```
prev_version = $(grep -o -E -m 1 'v[0-9]\.[0-9]+\.[0-9]' docs/source/release_notes.rst)
sed -n -e "/^\*\*Future Releases/,/^\*\*${prev_version}/p;/^\*\*${prev_version}/q" docs/source/release_notes.rst | sed '/^\.\. warning::$/d' | tail -n +2 | sed '$ d' | awk 'NF'
```

I checked the grep command but not sure if I substituted the previous version into sed properly 😄

Reply:
Very good point! Thanks for catching me here, I forgot that this was hard-coded in 🤦 

Yep, could grep for prev version. I opted for separating out the before vs the after.